website/docs/cdktf/concepts/constructs.mdx

```diff
@@ -291,8 +291,6 @@ You can search the Amazon Web Services (AWS) [Construct Hub](https://constructs.
 
 ~> **Note:** The AWS Adapter is in tech preview.
 
-## Publish Constructs
+## Write and Publish Constructs
 
-If you want to distribute construct libraries, we recommend writing the constructs in TypeScript and using `jsii` to generate native packages for all desired language targets. We recommend that open source projects aim to support all generally available languages.
-
-You can use the [package generator projen](https://github.com/projen/projen#getting-started) to scaffold a Github project with the necessary Github Actions workflows to build and publish your construct in multiple programming languages.
+You can write construct classes in any language and distribute them to other users. Refer to [Construct Design](/cdktf/develop-custom-constructs/construct-design) and [Construct Publishing and Distribution](/cdktf/develop-custom-constructs/publishing-and-distribution) for details.
```

Comment:
I made some changes to the intro as well (in the gdoc) :) would you mind pasting those in, too? :)